src/utils.js

```diff
@@ -178,7 +182,9 @@ function getFailedHooks(suite) {
  */
 function cleanSuite(suite, totalTestsRegistered) {
   suite.uuid = uuid.v4();
-  const failedHooks = _.map(getFailedHooks(suite), cleanTest);
+  const hooks = getFailedHooks(suite);
+  const beforeFailedHooks = _.map(hooks.before, cleanTest);
+  const afterFailedHooks = _.map(hooks.after, cleanTest);
```

Comment:
Ok i'll change that
About the  - ```hasBeforeFailedHooks``` and ```hasAfterFailedHooks```
Do you think it should be changed to ```hasBeforeHooks``` and ```hasAfterHooks``` as well ?